README.md

```diff
@@ -21,13 +21,72 @@ when building this AMI.
 
 ## Building the AMI
 
-A Makefile is provided to build the AMI, but it is just a small wrapper around
+A Makefile is provided to build the Amazon EKS Worker AMI, but it is just a small wrapper around
 invoking Packer directly. You can initiate the build process by running the
 following command in the root of this repository:
 
 ```bash
 make
 ```
+By default, Makefile will build Amazon EKS Worker AMI with Kubernetes binaries from these K8s versions
+1.16.15, 1.17.17, 1.18.20, 1.19.14, 1.20.10, 1.21.4
```

Comment:
This is no longer accurate since we're not generating 1.16 or 1.17 AMIs and are using diffferent kubelet versions for the others. 

Since this will keep happening in the future as we bring in new kubelet binaries, I'd recommend we change this line to something like "The Makefile chooses a particular kubelet binary to use per kubernetes version which you can [view here](https://github.com/awslabs/amazon-eks-ami/blob/master/Makefile)."

Reply:
Made the change